index.html

```diff
@@ -10,219 +10,268 @@
 </head>
 
 <body>
-    <div class="main-wrapper">
-        <header class="toolbar">
-            <div class="logo-place">
-                <a href="./index.html" class="my-logo hex">My Logo</a>
+    <section class="main-wrapper">
+            <header class="toolbar">
+                <div class="logo-place">
+                    <a href="./index.html" class="my-logo hex">My Logo</a>
+                </div>
+                <nav>
+                    <figure class="icons">
+                        <img src="images/Icons/Union.png" alt="Icons" />
+                    </figure>
+    
+                    <ul class="nav-bar-ul">
+                        <a href="#"><li>Portoflio</li></a>
+                        <a href="#"><li>About</li></a>
+                        <a href="#"><li>Contact</li></a>
+                      </ul>
```

Comment:
<img width="139" alt="image" src="https://user-images.githubusercontent.com/41163571/230602452-d43f5aa1-4074-455f-90f9-2b18e4d4f503.png">

- [ ] Kindly ensure that the nav elements follows the figma design, by removing the underline and making it match the font family and all to enable a pixel perfect design. 😄